### DISCUSSION TITLE
Close tab keybind

### DISCUSSION THREAD

**Discussion by Qtechinnovation** [ghostty-org/ghostty#3778] [Feature Requests, Ideas] [answered]
I noticed that https://ghostty.org/docs/config/keybind/reference does not mention an option to set a keybind to close a tab, but only to open one. I assumed that since opening a new tab by default uses the rather sensible ctrl+shift+t, that closing the tab would use ctrl+shift+w, but it does not. Is there an option to set a close tab keybind? I found a discussion about a person who forked ghostty to add one, but no interaction with the discussion, so I think this is warranted, especially for all the people who use keybinds as a way to increase productivity.

**Comment by mitchellh** [comment]
There is `close_surface` but that'll close splits first if you have them. Are you asking for a binding to close all splits and the current tab?

**Comment by mitchellh** [comment, marked as answer]
Approved #4331
